src/providers/krb5/krb5_ccache.c

```diff
@@ -529,7 +529,7 @@ errno_t sss_krb5_cc_verify_ccache(const char *ccname, uid_t uid, gid_t gid,
 
     mcred.client = princ;
     mcred.server = tgt_princ;
-    mcred.times.endtime = time(NULL);
+    mcred.times.endtime = time(NULL) & 0xFFFFFFFF;
```

Comment:
Hi,

please add a comment with a link to https://web.mit.edu/kerberos/krb5-latest/doc/appdev/y2038.html to make clear that libkrb5 is using 32bit time values for the time being.

bye,
Sumit

Reply:
Done.